src/QueryBuilder/ModelLoader.php

```diff
@@ -123,12 +123,14 @@ public function loadMissing($includePaths): self
      */
     public function loadCount($countable): self
     {
-        $counter = new CountableLoader(
-            $this->schema,
-            CountablePaths::cast($countable)
-        );
+        if ($countable) {
```

Comment:
We can actually avoid this unnecessary query in more cases than `$countable` being `null`, as there are multiple scenarios where `$countable` might resolve to an empty set of countable relationships.

Suggest amending to this:

```php
$paths = CountablePaths::cast($countable);

if ($paths->isNotEmpty()) {
    $counter = new CountableLoader($this->schema, $paths);
    $this->target->loadCount($counter->getRelations());
}
```